met-api/src/met_api/services/survey_service.py

```diff
@@ -47,6 +58,14 @@ def get_surveys_paginated(pagination_options: PaginationOptions, search_text='',
             'total': total
         }
 
+    @staticmethod
+    def _can_user_access_hidden_surveys(user_roles):
```

Comment:
can we invert the true/false logic here?

Theres a bit of text interpretation that doesn't make sense, 
Can user access hidden > returns TRUE means he cannot access it 
Reading the method name I'd assume that true means he can access it